host/hosted_os_adapter.h

```diff
@@ -15,6 +15,8 @@ typedef struct {
 /* 4 */   void*  (*_h_calloc)(size_t blk_no, size_t size);
 /* 5 */   void   (*_h_free)(void* ptr);
 /* 6 */   void*  (*_h_realloc)(void *mem, size_t newsize);
+          void*  (*_h_malloc_align)(size_t size, rt_size_t align);
+          void   (*_h_free_align)(void* ptr);
```

Comment:
`rt_size_t` -> `size_t`

For consistency, could add `/* 7 */` and `/* 8 */` here, although there is already a gap in the numbers.